@jmillan it fails to build on our server (jsut run `make` on `/tmp/mediasoup/worker`) :(
This is a fresh clone of mediasoup using the branch in this PR:

```
Found ninja-1.10.2.git.kitware.jobserver-1 at /tmp/kk/mediasoup/worker/out/pip/bin/ninja
/tmp/kk/mediasoup/worker/out/pip/bin/meson compile -C /tmp/kk/mediasoup/worker/out/Release/build -j 2 mediasoup-worker
INFO: autodetecting backend as ninja
INFO: calculating backend command to run: /tmp/kk/mediasoup/worker/out/pip/bin/ninja -C /tmp/kk/mediasoup/worker/out/Release/build -j 2 mediasoup-worker
ninja: Entering directory `/tmp/kk/mediasoup/worker/out/Release/build'
[1061/1333] Compiling C++ object subprojects/abseil-cpp-20230125.1/libabsl_base.a.p/absl_base_internal_cycleclock.cc.o
FAILED: subprojects/abseil-cpp-20230125.1/libabsl_base.a.p/absl_base_internal_cycleclock.cc.o
c++ -Isubprojects/abseil-cpp-20230125.1/libabsl_base.a.p -Isubprojects/abseil-cpp-20230125.1 -I../../../subprojects/abseil-cpp-20230125.1 -fdiagnostics-color=always -DNDEBUG -D_FILE_OFFSET_BITS=64 -std=c++14 -O3 -Wno-sign-compare -fPIC -pthread -MD -MQ subprojects/abseil-cpp-20230125.1/libabsl_base.a.p/absl_base_internal_cycleclock.cc.o -MF subprojects/abseil-cpp-20230125.1/libabsl_base.a.p/absl_base_internal_cycleclock.cc.o.d -o subprojects/abseil-cpp-20230125.1/libabsl_base.a.p/absl_base_internal_cycleclock.cc.o -c ../../../subprojects/abseil-cpp-20230125.1/absl/base/internal/cycleclock.cc
In file included from ../../../subprojects/abseil-cpp-20230125.1/absl/base/config.h:86:0,
                 from ../../../subprojects/abseil-cpp-20230125.1/absl/base/attributes.h:37,
                 from ../../../subprojects/abseil-cpp-20230125.1/absl/base/internal/cycleclock.h:48,
                 from ../../../subprojects/abseil-cpp-20230125.1/absl/base/internal/cycleclock.cc:23:
../../../subprojects/abseil-cpp-20230125.1/absl/base/policy_checks.h:57:2: error: #error "This package requires GCC 7 or higher."
 #error "This package requires GCC 7 or higher."
  ^~~~~
[1062/1333] Compiling C++ object subprojects/abseil-cpp-20230125.1/libabsl_base.a.p/absl_base_internal_low_level_alloc.cc.o
FAILED: subprojects/abseil-cpp-20230125.1/libabsl_base.a.p/absl_base_internal_low_level_alloc.cc.o
c++ -Isubprojects/abseil-cpp-20230125.1/libabsl_base.a.p -Isubprojects/abseil-cpp-20230125.1 -I../../../subprojects/abseil-cpp-20230125.1 -fdiagnostics-color=always -DNDEBUG -D_FILE_OFFSET_BITS=64 -std=c++14 -O3 -Wno-sign-compare -fPIC -pthread -MD -MQ subprojects/abseil-cpp-20230125.1/libabsl_base.a.p/absl_base_internal_low_level_alloc.cc.o -MF subprojects/abseil-cpp-20230125.1/libabsl_base.a.p/absl_base_internal_low_level_alloc.cc.o.d -o subprojects/abseil-cpp-20230125.1/libabsl_base.a.p/absl_base_internal_low_level_alloc.cc.o -c ../../../subprojects/abseil-cpp-20230125.1/absl/base/internal/low_level_alloc.cc
In file included from ../../../subprojects/abseil-cpp-20230125.1/absl/base/config.h:86:0,
                 from ../../../subprojects/abseil-cpp-20230125.1/absl/base/attributes.h:37,
                 from ../../../subprojects/abseil-cpp-20230125.1/absl/base/internal/low_level_alloc.h:31,
                 from ../../../subprojects/abseil-cpp-20230125.1/absl/base/internal/low_level_alloc.cc:20:
../../../subprojects/abseil-cpp-20230125.1/absl/base/policy_checks.h:57:2: error: #error "This package requires GCC 7 or higher."
 #error "This package requires GCC 7 or higher."
  ^~~~~
ninja: build stopped: subcommand failed.
Makefile:199: recipe for target 'mediasoup-worker' failed
make: *** [mediasoup-worker] Error 1
```

So... `#error "This package requires GCC 7 or higher."`

Amazing :(

--- a/worker/subprojects/abseil-cpp.wrap
+++ b/worker/subprojects/abseil-cpp.wrap
@@ -1,22 +1,19 @@
 [wrap-file]
-directory = abseil-cpp-20230125.1
-source_url = https://github.com/abseil/abseil-cpp/archive/20230125.1.tar.gz
-source_filename = abseil-cpp-20230125.1.tar.gz
-source_hash = 81311c17599b3712069ded20cca09a62ab0bf2a89dfa16993786c8782b7ed145
-patch_filename = abseil-cpp_20230125.1-1_patch.zip
-patch_url = https://wrapdb.mesonbuild.com/v2/abseil-cpp_20230125.1-1/get_patch
-patch_hash = a920ab28067e433d7ead2d564a4f511628f498f0723f7f94d19317877787ef39
-source_fallback_url = https://github.com/mesonbuild/wrapdb/releases/download/abseil-cpp_20230125.1-1/abseil-cpp-20230125.1.tar.gz
-wrapdb_version = 20230125.1-1
+directory = abseil-cpp-20220623.0
+source_url = https://github.com/abseil/abseil-cpp/archive/20220623.0.tar.gz
+source_filename = abseil-cpp-20220623.0.tar.gz
+source_hash = 4208129b49006089ba1d6710845a45e31c59b0ab6bff9e5788a87f55c5abd602
+patch_filename = abseil-cpp_20220623.0-2_patch.zip
+patch_url = https://wrapdb.mesonbuild.com/v2/abseil-cpp_20220623.0-2/get_patch
+patch_hash = d19cb16610d9310658a815ebcd87a9e2966aafbd57964341c0d1a3a3778c03b6
+wrapdb_version = 20220623.0-2
 
 [provide]
 absl_base = absl_base_dep
 absl_container = absl_container_dep
 absl_debugging = absl_debugging_dep
-absl_log = absl_log_dep
 absl_flags = absl_flags_dep
 absl_hash = absl_hash_dep
-absl_crc = absl_crc_dep
 absl_numeric = absl_numeric_dep
 absl_random = absl_random_dep
 absl_status = absl_status_dep

